Changelog.md

```diff
@@ -1,7 +1,8 @@
 ### 0.4.16 (unreleased)
 
 Features:
- * ABI JSON: Include new field ``statemutability`` with values ``view``, ``nonpayable`` and ``payable``.
+ * Introduce ``pure`` functions. The pureness is not enforced yet, use with care.
+ * ABI JSON: Include new field ``statemutability`` with values ``pure``, ``view``, ``nonpayable`` and ``payable``.
```

Comment:
any chance of making statemutability: `state_mutability`? Pure aesthetics I know. But might make it easier to read. Atleast a mixedCase or `-` of some kind.